src/Features/VisualBasicTest/EditAndContinue/StatementEditingTests.vb

```diff
@@ -4835,7 +4835,7 @@ End Class"
             edits.VerifySemanticDiagnostics()
         End Sub
 
-        <WpfFact(Skip:="https://github.com/dotnet/roslyn/issues/1212"), WorkItem("https://github.com/dotnet/roslyn/issues/1212")>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/1212"), WorkItem("https://github.com/dotnet/roslyn/issues/1212")>
```

Comment:
❓ All these tests are skipped. Are we expecting them to work without WpfFact?

Reply:
Yes, there should be no UI dependency.